src/ui/templates/results/verticalresults.hbs

```diff
@@ -69,11 +69,11 @@
   {{/if}}
 {{/inline}}
 
-{{#*inline "viewMore"}}
+{{#*inline "viewAll"}}
   {{#if _config.isUniversal}}
-    <div class="yxt-Results-viewMore">
-      <a class="yxt-Results-viewMoreLink" href="{{ verticalURL }}">
-        <div class="yxt-Results-viewMoreLabel">{{_config.viewMoreLabel}}</div>
+    <div class="yxt-Results-viewAll">
```

Comment:
This is interesting.

Anyone hoping to hide the View All link in older versions would have to change their target to viewAll if they wanted this styling to go away. Does this solve the initial problem?

Reply:
The initial problem being, Yext Answers was hiding viewAllLink to make sure it does not show on its promo card.

Reply:
eg. It will still look like this
![image](https://user-images.githubusercontent.com/32468132/84430109-2daff700-abf7-11ea-8802-e8db8e82629f.png)

Reply:
What I said was false. Yext Answers is not hiding viewAllLink, it was hiding the titleBar. Regardless of this change, if they wanted to hide the viewAllLink, they would need to add net new styling.

Reply:
hopefully partially addressed in #879